### ISSUE TITLE
Documentation of global variables

### ISSUE THREAD

**Issue by barakman** [OpenZeppelin/solidity-docgen#43]
As you know, the compiler implicitly creates a getter function for each global variable which is declared `public`.

Ideally, I'd like each one of these getter functions added to solidity-docgen's output (provided that I document them of course).

I assume that it might be a little difficult to actually display them as functions, because the solidity-parser (AST builder) adds them to the tree as variables.

So I would settle for having them displayed as variables.

For this purpose, I believe that you'd need to handle the `VariableDeclaration` node-type.

This means that in file `solidity.js`, you'd need to add:
1. Inside class `SolidityContract`:
```
    get ownVariables() {
        return this.astNode.nodes
            .filter(isVariableDeclaration)
            .filter(n => n.visibility === 'public')
            .map(n => new SolidityVariable(this, n));
    }
```
2. Outside class `SolidityContract`:
```
class SolidityVariable extends SolidityContractItem {
    constructor(contract, astNode) {
        super(contract, astNode);
        this.astNode = astNode;
    }
    get name() {
        return this.astNode.name;
    }
    get outputs() {
        return this.astNode.type;
    }
    get visibility() {
        return this.astNode.visibility;
    }
}
```
3. Outside class `SolidityContract`:
```
function isVariableDeclaration(node) {
    return node.nodeType === 'VariableDeclaration';
}
```
Thanks

**Comment by frangio** [comment]
Thanks for the suggestion! Agree that we need to support this. Feel free to submit a pull request with these changes.

**Comment by frangio** [comment]
Fixed in #44.
